default_config.yml

```diff
@@ -0,0 +1,15 @@
+language: en
+
+pipeline: supervised_embeddings
+
+policies:
+  - name: KerasPolicy
```

Comment:
should we add comments with the links to the docs for every component? That way users can learn from the suggested config

Reply:
The documentation is easy to find if you just enter the terms into the search bar in the docs.
Not sure if it's worth to have these comments then: it's annoying to implement (when dumping, we would need to look up again what the comments were, since they would not be stored in the `config` variable), and we would need to maintain them, i.e. update the links when the documentation changes

Reply:
Sounds good. That's also something we can do later if we think it might make sense.